index.html

```diff
@@ -274,6 +334,10 @@ <h2>Best Practices and Recommendations</h2>
         
         <p>For example, JavaScript internationalization [[ECMA-402]] and [[CLDR]] provide a "best fit" algorithm which can be tailored by implementers.</p>
         
+        <p class="advisement" id="ltli-keep-extensions"><a class="self" href="#ltli-keep-extensions">&#x200B;</a>Specifications SHOULD NOT restrict the length of language tags or permit or encourage the removal of extensions.</p>
+        
+        <p class="advisement" id="ltli-use-locale-fallback"><a class="self" href="#ltli-use-locale-fallback">&#x200B;</a>Specifications that define content selection based on language tags SHOULD </p>
```

Comment:
This sentence doesn't seem to be finished?

Reply:
Removed. Thanks!